notebooks/machine_learning/raw/tut8.ipynb

```diff
@@ -0,0 +1,479 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Classification\n",
+    "\n",
+    "Up to this point you've been using data to make real-valued predictions such as home prices. This type of modeling is called **regression**, hence the \"Regressor\" part of `RandomForestRegressor`. In general, regression means you are trying to predict real numbers like prices, failure rates, energy consumption, etc.\n",
+    "\n",
+    "Another common problem you'll see is making a choice between mutually exclusive outcomes. For example, spam detection is predicting if an email is \"spam\" or \"not spam\" based on the body text. Another common problem is identifying handwritten digits from images, choosing one of ten digits for any given image. This type of modeling is called **classification** because we are trying to choose one option from some set of classes.\n",
+    "\n",
+    "Classification is typically broken down further into binary and multiclass classification. Binary classification means there are two classes or conditions (\"spam\" or \"not spam\") while multiclass has more than two classes (ten digits). In general there are different approaches to the two types of classification, but most multiclass models will also work for binary problems.\n",
+    "\n",
+    "It's straightforward to build classification models using what you already know about scikit-learn. Instead of `RandomForestRegressor` for regression, we can use `RandomForestClassifier` ([documentation here](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html#sklearn-ensemble-randomforestclassifier)). \n",
```

Comment:
We used to link to documentation frequently. 

When watching users, they frequently first questioned whether that meant they are supposed to go check it out, which they felt conflicted about.

And then they did go to it, and it sometimes sent them on a yak-shaving trip around the internet trying to figure out what some of it meant.  

With our goal being for users to get to basic competency quickly, we're avoiding link-outs to docs.